queries.sql

```diff
@@ -68,3 +68,40 @@ SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
 
 
 SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+SELECT a.name 
+FROM animals a
+JOIN owners o ON a.owner_id = o.id
+WHERE o.full_name = 'Melody Pond';
+
+SELECT a.name 
+FROM animals a
+JOIN species s ON a.species_id = s.id
+WHERE s.name = 'Pokemon';
+
+SELECT o.full_name, a.name 
+FROM owners o
+LEFT JOIN animals a ON o.id = a.owner_id;
+
+SELECT s.name, COUNT(a.id) 
+FROM species s
+LEFT JOIN animals a ON s.id = a.species_id
+GROUP BY s.name;
+
+SELECT a.name 
+FROM animals a
+JOIN owners o ON a.owner_id = o.id
+JOIN species s ON a.species_id = s.id
+WHERE o.full_name = 'Jennifer Orwell' AND s.name = 'Digimon';
+
+SELECT a.name 
+FROM animals a
+JOIN owners o ON a.owner_id = o.id
+WHERE o.full_name = 'Dean Winchester' AND a.escape_attempts = 0;
```

Comment:
- [Optional] Dear @bohaz, According to the requirement, here we are asked to select animal information, in such case, we should use the `SELECT a.*` to select all the columns instead of just the name. Use such filtering (`SELECT a.name`) when specifically asked to choose just the name.